src/__integration_tests__/final-status-npg.integration.test.ts

```diff
@@ -15,31 +17,31 @@ describe("Unauthorized npg final status mapping tests", () => {
   const mockFlowWithExpectedResultMap = new Map([
     [
       "AUTHORIZATION_REQUESTED_NO_NPG_OUTCOME",
-      { title: "AUTHORIZATION REQUESTED WITH NO NPG OUTCOME", esito: "L'operazione è stata presa in carico" }
+      { title: "AUTHORIZATION REQUESTED WITH NO NPG OUTCOME", esito: "Attendi la conferma del pagamento" }
```

Comment:
what about take those description from the expected translation file key property?

something like
```code
import italianTranslations from "../translations/it/translations.json";

//and the test below will become
[
      "AUTHORIZATION_COMPLETED_WITH_NPG_AUTH_STATUS_EXECUTED",
      { title: "AUTHORIZATION COMPLETED WITH NPG AUTHORIZATION STATUS EXECUTED", esito: italianTranslations["GENERIC_ERROR"].title }
    ],
```

This way the test will continue to work properly even in the case of future translations update and here the correct outcome - translation key to be shown will be tested instead of performing an exact text match that will require all tests to be updated in case of any modification

Reply:
It's not so simple as it seems to be. Anyway this is a good idea, but to be handled in another PR.

Reply:
Yes, agree that it can be done into another PR :) 

> It's not so simple as it seems to be. 

I've tested this piece of code before suggesting it successfully, replacing all the `esito: "Si è verificato un errore imprevisto"` with `esito: italianTranslations["GENERIC_ERROR"].title`and executing tests checking for correct execution. 
Tested for just one case test it become a simple "replace all" operation for all the remaining string instances hardcoded into tests so it seems fair simple to me.
Do you see any other complication that I'm missing here?